src/commands/functions-secrets-destroy.ts

```diff
@@ -0,0 +1,35 @@
+import { Command } from "../command";
+import { logger } from "../logger";
+import { Options } from "../options";
+import { needProjectId } from "../projectUtils";
+import { destroySecretVersion } from "../gcp/secretManager";
+import { promptOnce } from "../prompt";
+
+export default new Command("functions:secrets:destroy <KEY>[@version]")
+  .description("Destroy a secret. Defaults to destroying the latest version.")
+  .withForce("Destroys a secret without confirmation.")
+  .action(async (key: string, options: Options) => {
+    const projectId = needProjectId(options);
+    let [name, version] = key.split("@");
+    if (!version) {
+      version = "latest";
+    }
+
+    if (!options.force) {
```

Comment:
nit: A command that always requires you to use "force" without even doing a dry run to see what the affects are seem off. Maybe something like requiring --force if the secret is currently in use? Or at least resolving what "version" is?

Reply:
If you don't specify force, users would get a prompt asking if they actually want to delete secret at the resolved version. If you do have --force, you don't get the prompt.

Not sure why you think this command always requires a force flag?

Reply:
I see what you mean by resolving the version though - that's a great idea.